Update/kage_006.txt

```diff
@@ -1303,7 +1303,7 @@ void main()
 			NULL, " I didn't mean to keep it a secret.", Line_WaitForInput);
 	ModPlayVoiceLS(4, 29, "ps3/s08/29/321200179", 256, TRUE);
 	OutputLine(NULL, "　ただ、…もう少しちゃんと目処がついたら、報告しようかなって思ってただけで…」",
-			NULL, " I just... thought it would be better not to mention it...\"", Line_Normal);
+			NULL, " I just... thought it would be better to wait until I was sure things were steady before telling...\"", Line_Normal);
```

Comment:
```suggestion
			NULL, " I just... thought it would be better to wait until I was sure things were steady before telling you...\"", Line_Normal);
```